hack/apitest.sh

```diff
@@ -5,7 +5,7 @@ BUILD_DIR="build"
 TEST_DIR="test"
 URL="https://hello.devops-art-factory.com"
 
-stacks=("artd" "artd-spot" "artd-mixed" "artd-without-targetgroup")
+stacks=("artd" "artd-spot" "artd-mixed" "artd-without-targetgroup" "artd-healthcheck-lb" "artd-with-exact-values" "artd-with-exact-values" "artd-with-classic-lb")
```

Comment:
`artd-with-exact-values` has duplicated!

Reply:
Fixed it :)